demos/source/aws_iot_demo_defender.c

```diff
@@ -62,6 +62,11 @@ static AwsIotDefenderError_t _startDefender( const char * pIdentifier,
                                              void * pNetworkCredentialInfo,
                                              const IotNetworkInterface_t * pNetworkInterface );
 
+/**
```

Comment:
Please take a look at the flow of the other demos, this demo should be similar.

Reply:
Ok will do and make changes if necessary.